docs/apis/http/oidc/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "OIDC"
+date: 2023-10-10T00:00:00+00:00
+weight: 21
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/apis/http/idp
+geekdocFilePath: _index.md
+geekdocCollapseSection: true
+---
+
+Infinite Scale can act as an identity provider (IdP) for OpenID Connect (OIDC) authentication .
+OIDC defines a discovery mechanism, called OpenID Connect Discovery,
+where an OpenID server publishes its metadata at a well-known URL, typically:
```

Comment:
Does the URL depends on server or it's the standard of OIDC?

Reply:
Searching Google for keywords: "OpenID Connect Discovery url" --> [swagger](https://swagger.io/docs/specification/authentication/openid-connect-discovery/)